src/surge-xt/osc/OpenSoundControl.cpp

```diff
@@ -346,6 +410,11 @@ void OpenSoundControl::send(std::string addr, std::string msg)
     }
 }
 
+void OpenSoundControl::sendError(std::string errorMsg)
+{
+    OpenSoundControl::send("/error", errorMsg);
```

Comment:
What does send error do if you don’t have an outbound osc port? Should it at least log to stdout in that case?

Reply:
It could do. Good idea.

Reply:
Should the stdout messages be wrapped in #if DEBUG?